src/state/reducer.js

```diff
@@ -79,55 +77,38 @@ const move = ({
 
   const previous: CurrentDrag = state.drag.current;
   const initial: InitialDrag = state.drag.initial;
-  const droppable: DroppableDimension = state.dimension.droppable[initial.source.droppableId];
+  const currentWindowScroll: Position = windowScroll || previous.windowScroll;
 
   const client: CurrentDragLocation = (() => {
     const offset: Position = subtract(clientSelection, initial.client.selection);
-    const center: Position = add(offset, initial.client.center);
 
     const result: CurrentDragLocation = {
-      selection: clientSelection,
       offset,
-      center,
-    };
-    return result;
-  })();
-
-  const page: CurrentDragLocation = (() => {
-    const offset: Position = subtract(pageSelection, initial.page.selection);
-    const center: Position = add(offset, initial.page.center);
-
-    const result: CurrentDragLocation = {
-      selection: pageSelection,
-      offset,
-      center,
+      selection: clientSelection,
+      center: add(offset, initial.client.center),
     };
     return result;
   })();
 
-  const scrollDiff: Position = subtract(droppable.scroll.initial, droppable.scroll.current);
-
-  const withinDroppable: WithinDroppable = {
```

Comment:
within droppable is gone. It was only used for get-drag-impact and the new function is much clearer